Library/Homebrew/extend/os/linux/dev-cmd/determine-test-runners.rb

```diff
@@ -0,0 +1,49 @@
+# typed: strict
+# frozen_string_literal: true
+
+class TestRunnerFormula
+  extend T::Sig
+
+  sig { returns(T::Array[TestRunnerFormula]) }
+  def dependents
+    @dependent_hash[ENV["HOMEBREW_SIMULATE_MACOS_ON_LINUX"].present?] ||= with_env(HOMEBREW_STDERR: "1") do
+      Utils.safe_popen_read(
+        HOMEBREW_BREW_FILE, "uses", "--formulae", "--eval-all", "--include-build", "--include-test", name
```

Comment:
Need to not pass `--eval-all` here unless it's being passed to the command being run, too.